lib/read-bitmap.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var fs = require('fs');
+var EventEmitter = require('events').EventEmitter;
+exports.emitter = new EventEmitter();
```

Comment:
by exporting the instance of `EventEmitter` here, you're preventing being able processing more than one of these at a time because multiple calls to `readBitmap` will trigger `done` on the same `emitter`.

a possible solution here would be to export a single thing from this file that can read a single bitmap and register its own events.